SWLOR.Game.Server/Data/Migrations/2018-12-02.1.sql

```diff
@@ -0,0 +1,45 @@
+﻿UPDATE dbo.Perk
+SET Description = 'Provides a bonus to repairs when using Electronic repair kits.'
+WHERE ID = 152
+UPDATE dbo.Perk
+SET Description = 'Provides a bonus to repairs when using armor repair kits'
+WHERE ID = 150
+UPDATE dbo.Perk
+SET Description = 'Provides a bonus to repairs when using weapon repair kits'
+WHERE ID = 149
+UPDATE dbo.PerkLevel
+SET Description = 'Gains +2 to the repair of armor when using an armor repair kit'
+WHERE ID = 2066
```

Comment:
Sorry, I should have mentioned some things about DB migrations. In tables where the ID is auto generated (PerkLevel is one of them) you have to do some weird code, since you can't guarantee the ID in your local copy will be the same ID on the server. 

An example of how you could do this is:

UPDATE dbo.PerkLevel
SET Description = 'Gain +2 to armor repair when using an armor repair kit.'
WHERE PerkID =150 AND Level = 1

Perk IDs are manually entered so we're positive they won't change. PerkLevel IDs on the other hand are not, so we have to be explicit. 

This post has some info on finding out whether a column is an IDENTITY: https://stackoverflow.com/questions/87747/how-do-you-determine-what-sql-tables-have-an-identity-column-programmatically

Reply:
Ah okay I'll do what you did with the example then